### PR TITLE
Attach chromium protocol commands to msedge sessions

### PR DESCRIPTION
**Pull request by rhorm**
## Proposed changes

Allows Chromium Edge to use Chromium Protocol commands.   Also changed isChrome to isChromium since Chrome and Edge both use it.  

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

- [x ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

My first contribution!  Take it easy on me ;-)

### CODE REVIEW

File: packages/wdio-utils/src/envDetector.js [hunk 1]
```diff
@@ -42,12 +42,12 @@ function isW3C (capabilities) {
...
  */
-function isChrome (caps) {
+function isChromium (caps) {
     if (!caps) {
         return false
     }
     return (
-        Boolean(caps.chrome) ||
+        Boolean(caps.chrome || caps.msedge) ||
         Boolean(caps['goog:chromeOptions'])
```
Comment (ronhorm) [inline packages/wdio-utils/src/envDetector.js:51]: Not sure why check for chromeOptions is needed.  Left it alone

**Comment by rhorm** [comment]
not sure what is going on with CLA.   its saying is not signed here in the comments but, when I open to sign it says already signed.

**Comment by christian-bromann** [comment]
> not sure what is going on with CLA. its saying is not signed here in the comments but, when I open to sign it says already signed.

@rhorm please make sure the email you use to make the commit is the same you signed the CLA with.

**Comment by rhorm** [comment]
I already signed with the wrong one darn it.   should I resign or close this PR and resubmit?

**Comment by christian-bromann** [comment]
> should I resign or close this PR and resubmit

Let's do that.

**Pull request closed without merging**